### PR TITLE
Run go mod vendor and update update-deps.sh

### PR DESCRIPTION
**Pull request by ImJasonH**
This updates the cached deps in `vendor/` and update `update-deps.sh` to ensure this is done when new deps are updated.

Also fix a lint error in `cmd/gcs-uploader/main.go`

cc @dprotaso

### CODE REVIEW

File: gcs-fetcher/cmd/gcs-uploader/main.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func main() {
 	}
 	bucket, object, generation, err := common.ParseBucketObject(*location)
 	if err != nil {
-		log.Fatalln("parsing location from %q: %v", *location, err)
+		log.Fatalf("parsing location from %q: %v", *location, err)
```
Comment (dprotaso) [inline gcs-fetcher/cmd/gcs-uploader/main.go]: \n at the end

Comment (ImJasonH) [reply gcs-fetcher/cmd/gcs-uploader/main.go]: `log` methods already append a newline at the end. I'm not sure why both `log.Print` and `log.Println` exist...

Comment (dprotaso) [reply gcs-fetcher/cmd/gcs-uploader/main.go]: cool

Comment (dprotaso) [reply gcs-fetcher/cmd/gcs-uploader/main.go]: lgtm

**Comment by dprotaso** [comment]
@codrienne it's good to merge without the `\n` change.

**Pull request merged**